manager/Makefile

```diff
@@ -59,7 +59,7 @@ wait_for_manager: $(TOOLBIN)/kubectl
 .PHONY: run-integration-tests
 run-integration-tests: export DOCKER_HOSTNAME?=localhost:5000
 run-integration-tests: export DOCKER_NAMESPACE?=fybrik-system
-run-integration-tests: export USE_WKC_CONTROLLER?=true
+run-integration-tests: export USE_WKC_CONTROLLER?=false
```

Comment:
If I understand correctly, the intent is to avoid running e2e test in any but mockup environment. Can perhaps a name like USE_MOCKUP_CONNECTOR be used (which will be true by default)? If any other connector is used (Egeria, Katalog) the e2e test will fail since "s3/allow-dataset" is not a valid name of the asset.

Reply:
Yes, I can make this more generic.  Do we want it to be true by default, though?

Reply:
We want the tests to run in CI with mockup controllers, there it can be set explicitly, just like USE_EXISTING_CONTROLLER. 
The default is not important, can even be left empty as with USE_EXISTING_CONTROLLER. 
Suggest to rename to USE_EXISTING_CONNECTORS or something alike.
BTW, if I am not mistaken, USE_EXISTING_CLUSTER is not used in the code and can be removed...

Reply:
Thanks - this change was made